### ISSUE TITLE
Translate or Transcribe?

### ISSUE THREAD

**Issue by oldsongsz** [SevaSk/ecoute#60]
class APIWhisperTranscriber:
    def get_transcription(self, wav_file_path):
        try:
            with open(wav_file_path, "rb") as audio_file:
                result = openai.Audio.translate("whisper-1", audio_file)
        except Exception as e:
            print(e)
            return ''
        return result['text'].strip()

result = openai.Audio.translate("whisper-1", audio_file) # here should be openai.Audio.transcribe ?

**Comment by SevaSk** [comment]
Good catch! Just pushed a change to make this transcribe. Since whisper API auto detects language this change also makes it so the --api flag works with most languages!

**Comment by oldsongsz** [comment]
lots of fun.
